web-client/src/presenter/actions/setPractitionerClosedCasesPageAction.ts

```diff
@@ -1,11 +1,19 @@
 import { state } from '@web-client/presenter/app.cerebral';
 
 export const setPractitionerClosedCasesPageAction = ({
+  get,
   props,
   store,
 }: ActionProps<{
   pageNumber: number;
 }>) => {
   const { pageNumber } = props;
-  store.set(state.practitionerDetail.closedCaseInfo?.currentPage, pageNumber);
+  const closedCaseInfo = get(state.practitionerDetail.openCaseInfo);
+
+  if (!closedCaseInfo) {
+    throw new Error(
+      'Attempting to set state.practitionerDetail.closedCaseInfo.currentPage without first setting up state.practitionerDetail.closedCaseInfo',
+    );
+  }
+  store.set(state.practitionerDetail.closedCaseInfo!.currentPage, pageNumber);
```

Comment:
If there is a better way to handle this, I'd be happy to hear it! Cerebral did not like the optional chaining I had before.